README.md

```diff
@@ -1 +1,76 @@
-# staging-branches
\ No newline at end of file
+# staging-branches
+
+`staging-branches` is a repo hosting the branches configuration for [commcare-hq](https://github.com/dimagi/commcare-hq/)'s staging environment
+
+## Workflow
+
+* Add the branch you'd like to deploy to staging to [the yaml file](https://github.com/dimagi/staging-branches/blob/main/commcare-hq-staging.yml) in this repository. You may use the github UI or edit it however you choose.
+* Commit your changes directly to master.  Don't worry about a detailed commit message unless you're doing something unusual.
+* From the root of the `commcare-hq` repository, run `./scripts/rebuildstaging`.  This will build a branch called `autostaging` that contains all branches specified in this file.
+  * To check if your nelwy added branch causes any conflicts before pushing to autostaging, run:
+    ```
+    $ scripts/rebuildstaging --no-push
+    ```
+  * To rebuild autostaging and push those changes to origin (necessary for deploy), run:
+     ```
+     $ scripts/rebuildstaging
+     ```
+* After rebuilding the autostaging branch, you need to deploy the new branch to staging.
+  ```
+  $ commcare-cloud --control staging deploy
+  ##### OR #####
+  $ scripts/rebuildstaging --deploy (NOTE: `commcare-cloud` must be available in your shell)
+  ```
+
+## RESOLVING BRANCH CONFLICTS
+
+First, determine where the conflict lies.
```

Comment:
Would it make sense to add a note here about performing these git operations in the repo that has a conflict? That is, not in this _staging-branches_ repo.

Reply:
oh yeah, good point, I'll edit the file directly